apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -312,26 +354,80 @@ class CensusForm extends Component {
   }
 
   render() {
-    const showFollowUp = this.state.showFollowUp;
-    const showPledge = this.state.showPledge;
-    const submission = this.state.submission;
-    const errors = this.state.errors;
-    const showErrorMsg = !!(errors.email || errors.topics || errors.frequency || errors.school || errors.role || errors.hoc || errors.afterSchool || errors.tenHours || errors.twentyHours);
+    console.log("STATE:", this.state);
+    const { showFollowUp, showPledge, submission, errors } = this.state;
+    const showErrorMsg = !!(errors.email || errors.topics || errors.frequency || errors.school || errors.role || errors.hoc || errors.afterSchool || errors.tenHours || errors.twentyHours || errors.country || errors.nces);
+    const US = submission.country === "United States";
```

Comment:
The `"United States"` string literal has been used at least twice in this file - let's extract it into a constant.